server/src/main/java/dev/slimevr/config/CurrentVRConfigConverter.java

```diff
@@ -18,188 +19,212 @@ public ObjectNode convert(
+						// Filter only number values because other types would
+						// be
+						// stuff that didn't get migrated correctly before
+						if (node.getValue().isNumber()) {
+							offsetsNode.set(node.getKey(), node.getValue());
+						}
+					}
+
+					// Fix calibration wolf typos
+					offsetsNode.set("shouldersWidth", bodyNode.get("shoulersWidth"));
+					offsetsNode.set("shouldersDistance", bodyNode.get("shoulersDistance"));
+					offsetsNode.remove("shoulersWidth");
+					offsetsNode.remove("shoulersDistance");
 					skeletonNode.set("offsets", offsetsNode);
 					modelData.set("skeleton", skeletonNode);
+					modelData.remove("body");
 				}
 			}
-		}
-		if (version < 6) {
-			// Migrate controllers offsets to hands offsets
-			ObjectNode skeletonNode = (ObjectNode) modelData.get("skeleton");
-			if (skeletonNode != null) {
-				ObjectNode offsetsNode = (ObjectNode) skeletonNode.get("offsets");
-				if (offsetsNode != null) {
-					offsetsNode.set("handDistanceY", offsetsNode.get("controllerDistanceY"));
-					offsetsNode.set("handDistanceZ", offsetsNode.get("controllerDistanceZ"));
+			if (version < 3) {
+				// Check for out-of-bound filtering amount
+				ObjectNode filtersNode = (ObjectNode) modelData.get("filters");
+				if (filtersNode != null && filtersNode.get("amount").floatValue() > 2f) {
+					filtersNode.set("amount", new FloatNode(0.2f));
 				}
 			}
-		}
-		if (version < 7) {
-			// Chest, hip, and elbow offsets now go the opposite direction
-			ObjectNode skeletonNode = (ObjectNode) modelData.get("skeleton");
-			if (skeletonNode != null) {
-				ObjectNode offsetsNode = (ObjectNode) skeletonNode.get("offsets");
-				if (offsetsNode != null) {
-					if (offsetsNode.get("chestOffset") != null) {
+			if (version < 4) {
+				// Change mountingRotation to mountingOrientation
+				ObjectNode oldTrackersNode = (ObjectNode) modelData.get("trackers");
+				if (oldTrackersNode != null) {
+					var trackersIter = oldTrackersNode.iterator();
+					var fieldNamesIter = oldTrackersNode.fieldNames();
+					ObjectNode trackersNode = nodeFactory.objectNode();
+					String fieldName;
+					while (trackersIter.hasNext()) {
+						ObjectNode node = (ObjectNode) trackersIter.next();
+						fieldName = fieldNamesIter.next();
+						node.set("mountingOrientation", node.get("mountingRotation"));
+						node.remove("mountingRotation");
+						trackersNode.set(fieldName, node);
+					}
+					modelData.set("trackers", trackersNode);
+				}
+			}
+			if (version < 5) {
+				// Migrate old skeleton offsets to new ones
+				ObjectNode skeletonNode = (ObjectNode) modelData.get("skeleton");
+				if (skeletonNode != null) {
+					ObjectNode offsetsNode = (ObjectNode) skeletonNode.get("offsets");
+					if (offsetsNode != null) {
+						// torsoLength, chestDistance and waistDistance become
+						// chestLength, waistLength and hipLength.
+						float torsoLength = 0, chestDistance = 0, waistDistance = 0;
```

Comment:
This isn't very desirable to set missing values to `0`, can we do default lengths here? Neither solution is perfect, but if anything goes wrong, I'd rather it goes the least wrong.